packages/renderer/src/lib/dialogs/CommandPalette.svelte

```diff
@@ -59,10 +68,15 @@ let searchOptions: SearchOption[] = $derived([
   { text: 'Go to', shortCut: [`${modifierC}F`] },
 ]);
 let searchOptionsSelectedIndex: number = $state(0);
+let imageItems: ImageInfo[] = $derived($imagesInfos);
+let containerItems: ContainerInfo[] = $derived($containersInfos);
+let podItems: PodInfo[] = $derived($podsInfos);
+let volumeItems: VolumeInfo[] = $derived($volumeListInfos.map(info => info.Volumes).flat());
 
-let commandInfoItems: CommandInfo[] = $state([]);
+let commandInfoItems: CommandInfo[] = $derived($commandsInfos);
 let documentationItems: DocumentationInfo[] = $state([]);
-let globalContext: ContextUI;
+let goToItems: GoToInfo[] = $derived(createGoToItems(imageItems, containerItems, podItems, volumeItems));
```

Comment:
```suggestion
let goToItems: GoToInfo[] = $derived(createGoToItems($imageItems, $containerItems, $podItems, $volumeItems));
```
would works the same way without wrapping it in $derived